src/shared/daemon-util.c

```diff
@@ -74,3 +75,18 @@ int notify_push_fdf(int fd, const char *format, ...) {
 
         return notify_push_fd(fd, name);
 }
+
+int notify_reloading_full(const char *status) {
+        int r;
+
+        r = sd_notifyf(/* unset_environment = */ false,
+                       "RELOADING=1\n"
+                       "MONOTONIC_USEC=" USEC_FMT
+                       "%s%s",
+                       now(CLOCK_MONOTONIC),
+                       status ? "\nSTATUS=" : "", strempty(status));
+        if (r < 0)
+                return log_warning_errno(r, "Failed to notify service manager for reloading status: %m");
```

Comment:
warning level means that while previously errors were silent in all components, they are now logged, so switch to debug to keep it as-is?